installer/pkg/config-generator/tls.go

```diff
@@ -36,10 +37,12 @@ func (c *ConfigGenerator) GenerateTLSConfig(clusterDir string) error {
 	var caCert *x509.Certificate
 	var kubeCAKey *rsa.PrivateKey
 	var kubeCACert *x509.Certificate
+	var certValidity time.Duration
 	var err error
 	var baseAddress string
 
 	baseAddress = c.getBaseAddress()
+	certValidity = 26280 * time.Hour
```

Comment:
Why 365? Since this is not a calculated number but rather a fixed value, it should be made into a `const` at the top of the package. The const should be named something like:
```go
const (
    validityThreeYears = time.Hour * 24 * 365 * 3
)
```
this way it is extremely clear what the value is and how the number was derived. Plus it is only calculated one time, at compilation.

See bootkube for reference https://github.com/kubernetes-incubator/bootkube/blob/master/pkg/tlsutil/tlsutil.go#L18

Reply:
thanks! fixed